zib/StructureDefinitions/ext-Dosage-AsNeededFor.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+   <id value="ext-Dosage-AsNeededFor"/>
+   <text>
+    <status value="empty" /><div xmlns="http://www.w3.org/1999/xhtml">No narrative is provided for definitional resources. A human-readable rendering can be found in the implementation guide(s) where this resource is used.</div></text>
+   <url value="http://nictiz.nl/fhir/StructureDefinition/ext-Dosage-AsNeededFor"/>
+   <name value="ExtDosageAsNeededFor"/>
+   <title value="ext Dosage AsNeededFor"/>
+   <status value="active"/>
+   <publisher value="Nictiz"/>
+   <contact>
+      <name value="Nictiz"/>
+      <telecom>
+         <system value="url"/>
+         <value value="https://www.nictiz.nl"/>
+         <use value="work"/>
+      </telecom>
+   </contact>
+   <description value="Pre-adopt of the `Dosage.asNeededFor` concept from the FHIR R5 Dosage data type. In R4, `Dosage.asNeededCodeableConcept` has a maximum cardinality of 1. Additional 'asNeeded' data can be represented by adding this extension to Dosage."/>
```

Comment:
Maybe we can check if we can use the cross version extensions from FHIR for this purpose? I believe in the meantime, the specification for this has been improved: https://hl7.org/fhir/R5/versions.html#extensions perhaps these extensions are also available. We can at least check.

Reply:
Nice! Mm, see Teams chat. Perhaps ask for an update? It has been a while. Decide on monday with the team what our approach would be regarding the canonicals.

Reply:
Ok, it will likely still take quite a while before something usefull is published. Created https://nictiz.atlassian.net/browse/ZIBFHIR-230 to see if we can do a find and replace before final publication.